### ISSUE TITLE
Overloading `Base.sqrt` should re-run cells that call `sqrt`

### ISSUE THREAD

**Issue by fonsp** [fonsp/Pluto.jl#1019]
https://user-images.githubusercontent.com/6933510/112028853-eb79f580-8b38-11eb-9e93-d9a0fe070715.mov

**Comment by Pangoraw** [comment]
If we offload more and more work on the notebook process, we could be able to do something like this to retrieve the module for a method:
```julia
julia> sqrt_name = :sqrt
:sqrt

julia> Core.eval(Main, :(Symbol(typeof($sqrt_name).name.mt.module))) # on the notebook process
:Base
```

**Comment by fonsp** [comment]
This is true, and it sounds cool, but in this case, I would say that defining a method for `A.B.x` should re-run any reference to `x` or `B.x`, as a "greedy" solution.
